tests/modules/test_ApacheBenchmark.py

```diff
@@ -0,0 +1,119 @@
+import logging
+import time
+from os import path
+
+from bzt.modules.apachebench import ApacheBenchExecutor, TSVDataReader
+from tests import BZTestCase
+from tests.mocks import EngineEmul
+from bzt.utils import EXE_SUFFIX
+
+
+TOOL_NAME = 'ab' + EXE_SUFFIX
+
+
+def get_res_path(resource):
+    return path.join(path.dirname(__file__), '..', 'apachebench', resource)
+
+
+class TestApacheBenchExecutor(BZTestCase):
+    def test_iter(self):
+        "Ensures that executor doesn't fail with minimal configuration."
+        obj = ApacheBenchExecutor()
+        obj.engine = EngineEmul()
+        obj.settings.merge({
+            "path": get_res_path(TOOL_NAME),})
+        obj.execution.merge({
+            "concurrency": 2,
+            "iterations": 3,
+            "scenario": {
+                "requests": ["http://blazedemo.com"]
+            }
+        })
+        obj.prepare()
+        obj.startup()
```

Comment:
good walkthrough test would also call check , shutdown and post-process methods. Just in case.